website/client/assets/svg/quest-background-border.svg

```diff
@@ -6,7 +6,7 @@
         <path id="d" d="M0 70h10v10H0z"/>
         <path id="e" d="M414 70h10v10h-10z"/>
     </defs>
-    <g fill="none" fill-rule="evenodd">
+    <g fill="#F9F9F9" fill-rule="evenodd">
```

Comment:
just fyi, if you remove the fill attribute then you can style the icon with a css `color: XXXX`